gammapy/modeling/models/temporal.py

```diff
@@ -617,6 +618,28 @@ def from_table(cls, table, filename=None):
 
         return cls(m, t_ref=t_ref, filename=filename)
 
+    @classmethod
+    def from_powerspectrum(cls, powerspectrum, map, **kwargs):
+        """Create a template model from a power spectrum using the Timmer algorithm.
+
+        Parameters
+        ----------
+        powerspectrum: `~numpy.ndarray`
+            Array representing the power spectrum from which to derive the model.
+        map:  `RegionNDMap`
+            Map containing the desired geometry of the model.
```

Comment:
I am not sure that it will work correctly if there are gaps in the data. So a map with a `TimeMapAxis` might be an issue

Reply:
For our fields, we will have time gaps in general. So, I am wondering if instead one could not use a list of GTI as input.